src/App.tsx

```diff
@@ -20,12 +30,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={event => setQuery(event.target.value)}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={filterMovies()} />
```

Comment:
-create a `visibleMovies` variable containing filtered movies;
Instead you created a function) It works, but doesn't match the task requirments. Let's see what mentor will say about this.

Reply:
agree